CMakeLists.txt

```diff
@@ -553,6 +641,7 @@ if(ENABLE_SPNAV)
 endif()
 
 add_executable(OpenSCAD ${Sources} ${RESOURCE_FILES})
+set_property(TARGET OpenSCAD PROPERTY CMAKE_CXX_STANDARD 11)
```

Comment:
Do we need this when we set CMAKE_CXX_STANDARD to 11 globally?

Reply:
I'm pretty sure it wasn't taking the global setting for some reason.  

I will try again with that commented out and see if its still an issue.  I did an update/upgrade on vcpkg today and it seemed to mess up my harfbuzz install or something, I'm trying to reinstall now (in order to reinstall I had to remove it plus dependencies, namely qt5, so its taking a while to rebuild).

Reply:
Looks like it works without it, removed now.